src/cli/.azuredevops/stage-publish-cli.yml

```diff
@@ -0,0 +1,56 @@
+parameters:
+- name: pathToSrc
+  type: string
+  default: '$(Build.SourcesDirectory)/src'
+
+jobs:
+- job: OnWindows_Build
+  pool:
+    vmImage : windows-2022
```

Comment:
nitpick: don't we have already a variable for `windows-2022`

Reply:
Nice catch! Copy and past error.